Makefile

```diff
@@ -0,0 +1,50 @@
+BINDIR = bin
+SRC_DIRS = pkg
+GOFILES = $(shell find $(SRC_DIRS) -name '*.go' | grep -v bindata)
+
+BUILD_CMD ?= docker build
+
+# Image URL to use all building/pushing image targets
+IMG ?= aws-account-operator:latest
+
+# Look up distro name (e.g. Fedora)
+DISTRO ?= $(shell if which lsb_release &> /dev/null; then lsb_release -si; else echo "Unknown"; fi)
+
+OPERATOR_IMAGE_URI=$(IMAGE_REGISTRY)/$(IMAGE_REPOSITORY)/$(IMAGE_NAME):v$(VERSION_FULL)
+
+BINFILE=build/_output/bin/$(OPERATOR_NAME)
+MAINPACKAGE=./cmd/manager
+GOENV=GOOS=linux GOARCH=amd64 CGO_ENABLED=0
+GOFLAGS=-gcflags="all=-trimpath=${GOPATH}" -asmflags="all=-trimpath=${GOPATH}"
+
+TESTTARGETS := $(shell go list -e ./... | egrep -v "/(vendor)/")
+TESTOPTS := 
+
+default: gobuild
+
+.PHONY: clean
+clean:
+	rm -rf ./build/_output
+
+.PHONY: gotest
+gotest:
+	go test $(TESTOPTS) $(TESTTARGETS)
+
+.PHONY: gocheck
+gocheck: ## Lint code
+	gofmt -s -l $(shell go list -f '{{ .Dir }}' ./... ) | grep ".*\.go"; if [ "$$?" = "0" ]; then gofmt -s -d $(shell go list -f '{{ .Dir }}' ./... ); exit 1; fi
+	go vet ./cmd/... ./pkg/...
+
+.PHONY: gobuild
+gobuild: gocheck gotest ## Build binary
+	${GOENV} go build ${GOFLAGS} -o ${BINFILE} ${MAINPACKAGE}
+
+# Build the docker image
+.PHONY: docker-build
+docker-build:
+	$(BUILD_CMD) -t ${IMG} build/.
+
+# Push the docker image
+.PHONY: docker-push
+docker-push:
+	$(DOCKER_CMD) push ${IMG}
```

Comment:
maybe you want a default for `DOCKER_CMD` ?

Reply:
@jmelis here? https://github.com/openshift/aws-account-operator/pull/39/files#diff-b67911656ef5d18c4ae36cb6741b7965R5

Reply:
but that's BUILD_CMD not DOCKER_CMD?

Reply:
ha. My eyes need a rest you are a right sorry I'll fix!

Reply:
ah I see, it's a typo it should be s/DOCKER_CMD/BUILD_CMD/

Reply:
@jmelis it should be different since the one is `docker build` and the other is `docker push` I just got mixed up. All fixed now.